baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -14,22 +14,26 @@ Header name: `Baggage`
 
 # Header Content
 
-This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule in <a data-cite='!RFC5234#appendix-B.1'>appendix B.1 for RFC5234</a>. It also includes the `OWS` rule (optional whitespace) from <a data-cite='!RFC7230#whitespace'>RFC7230 section 3.2.3</a>.
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]].
```

Comment:
What is `[[!XXX]]` syntax?

Reply:
> Likewise, in order to insert a normative reference, look for its code in the database and use the same syntax but with a "!" in front of the code: [[!CODE]]. This will produce a result like the following: [ELEMENTTRAVERSAL].

https://dev.w3.org/2008/video/mediaann/ReSpec.js/documentation.html